src/app/views/orders/order.controller.coffee

```diff
@@ -30,6 +36,16 @@
           vm.user = response.data.plain()
       )
 
+  vm.formatJson = (json) ->
```

Comment:
@hedudelgado @alexnoox @fgourichon Seriously? Don't we have better ways to generate a display from a json payload??

Reply:
The only other alternative I have in mind is creating a component that generates dynamically an HTML template depending on the JSON keys and values.

Reply:
Pretty much anything that does not involve string replace operations would be better.

So yes, generating HTML for a web page sounds like a good idea.

Reply:
I addressed this issue in this PR: https://github.com/maestrano/mnoe-admin-panel/pull/113